extensions/ims/user.schema.json

```diff
@@ -36,6 +55,11 @@
     },
     {
       "$ref": "#/definitions/user"
+    },
+    {
+      "$ref": "#/definitions/publicprofile"
     }
-  ]
+  ],
+  "required" : ["xdm:identityProvider", "id", "type"],
+  "meta:status": "experimental"
```

Comment:
Hi @lrosenthol 
I wish to mark the schema as experimental and specify some required fields. I think the schema is not used at least in this repo. But, it is not marked experimental, unfortunately. Since it is not marked experimental, I was hesitant to change existing properties like changing id to xdm:id. Should I make the change?